client/signup/reskinned-processing-screen/index.jsx

```diff
@@ -62,15 +62,17 @@ export default function ReskinnedProcessingScreen( { flowName, hasPaidDomain } )
 					'--progress': ! hasStarted ? /* initial 10% progress */ 0.1 : progress,
 				} }
 			/>
-			<p className="reskinned-processing-screen__progress-numbered-steps">
-				{
-					// translators: these are progress steps. Eg: step 1 of 4.
-					sprintf( __( 'Step %(currentStep)d of %(totalSteps)d' ), {
-						currentStep: currentStep + 1,
-						totalSteps,
-					} )
-				}
-			</p>
+			{ flowName !== 'launch-site' && (
```

Comment:
@StefanNieuwenhuis, it this fixing https://github.com/Automattic/wp-calypso/issues/54118 ? 
If yes, can you update the issue & this PR description to reflect that?

Also, I guess we can make it more generic:
```suggestion
			{ totalSteps > 1 && (
```

Reply:
> @StefanNieuwenhuis, it this fixing #54118 ?

It is fixing it but it was supposed to be fixed in https://github.com/Automattic/wp-calypso/pull/53993.
So I reverted it and applied your suggestion in f3725f09575104181cb74af29ff3d33622b91dd1.